### PR TITLE
Prepare v0.12.0 release

### PR DESCRIPTION
**Pull request by arminru**

### CODE REVIEW

File: semantic-conventions/CHANGELOG.md
Comment (arminru) [inline semantic-conventions/CHANGELOG.md:9]: @open-telemetry/technical-committee @open-telemetry/build-tools-approvers
Do you think we should wait for #79 to be merged and release both changes at once or release v0.12 now and v0.13 with metrics support soon-ish?

Comment (arminru) [reply semantic-conventions/CHANGELOG.md:9]: As discussed on the OTel spec call: No objections to releasing v0.12.0 right away as-is.

**Review by codeboten — APPROVED** [review]

**Review by reyang — APPROVED** [review]

**Pull request merged**
